addons/pos_online_payment_self_order/tests/test_self_order_frontend.py

```diff
@@ -3,10 +3,16 @@
 from unittest.mock import patch
 
 import odoo.tests
+from odoo.addons.pos_restaurant.tests import test_frontend
```

Comment:
Small nitpicking for consistency + see comment below
```suggestion
from odoo.addons.pos_restaurant.tests.test_frontend import TestFrontendCommon
```